tests/testthat/_snaps/resp-body.md

```diff
@@ -11,17 +11,17 @@
     Code
       request_test("/xml") %>% req_perform() %>% resp_body_json()
     Condition
-      Error in `check_content_type()`:
+      Error in `resp_body_json()`:
       ! Unexpected content type 'application/xml'
-      Expecting 'application/json'
-      Or suffix '+json'
+      i Expecting 'application/json' or 'application/<subtype>+json'
```

Comment:
It is a bit more verbose than "or suffix +json" but might be clearer. What do you think?

Reply:
I like it!